man/machinectl.xml

```diff
@@ -553,6 +553,17 @@
         <xi:include href="version-info.xml" xpointer="v206"/></listitem>
       </varlistentry>
 
+       <varlistentry>
+        <term><option>--value</option></term>
+
+        <listitem><para>When printing properties with <command>show</command>, only print the value,
+        and skip the property name and <literal>=</literal>.</para>
+
+        <xi:include href="version-info.xml" xpointer="v230"/></listitem>
+      </varlistentry>
+
+      <xi:include href="timedatectl.xml" xpointer="option-P"/>
```

Comment:
I moved the main text to standard-options.xml now. But I kept the include from timedatectl to machinectl, because that also has the version string, so it's just one line.